nubilanding/src/components/header/styles.module.scss

```
@@ -0,0 +1,52 @@
+$color-fondo: #eeeeee;
+$color-letra: #214960;
+$color-boton: #567efb;
+
+header {
+  display: flex;
+  width: 100%;
+  overflow: hidden;
+  background-color: $color-fondo;
+  flex-direction: row;
+  justify-content: flex-end;
+  text-align: center;
+}
+
+nav ul {
+  display: flex;
+  
+}
+
+.registrate{
```

Espacio entre textos y símbolos:
```suggestion
.registrate {
```